docs/pages/clients/installation.md

```diff
@@ -72,13 +72,23 @@ You're now ready to start developing your project with the Development Client.
 
 When you start your project on iOS, the metro bundler will be started automatically. This behavior might not be ideal when you want to use `expo start`. Our recommended solution is to remove the `Start Packager` action from building scripts. To do that you need to open the Xcode, go to `Project settings` > `Build Phases` and remove the `Start Packager` action.
 
+<img src="/static/images/client/remove_start_packager.png" style={{maxWidth: "100%" }}/>
+
 ### Add better error handlers
 
-Sometimes, for certain types of errors, we can provide more helpful error messages than the ones that ship by default with React Native. To turn this feature on, you need to add following lines to your `index.js`:
+Sometimes, for certain types of errors, we can provide more helpful error messages than the ones that ship by default with React Native. To turn this feature on, you need to
+
+- Create a file with `.fx.js` or `.fx.ts` extension (for example `DevLauncherSetUpErrorHandlers.fx.js`) which contains following lines:
```

Comment:
Could we have this file be inside of expo-dev-client and imported in the main file of expo-dev-client, so that all users need to do is add `import 'expo-dev-client';` in their index.js?

Reply:
Yes, we can do it ;)